tests/account-migration.test.ts

```
@@ -0,0 +1,43 @@
+import { describe, expect, test } from "bun:test";
+import { prisma } from "@/utils/prisma";
+import { ADMIN_ACCOUNT_ID } from "@/utils/constants";
```

ESLint

[error] Replace `prisma·}·from·"@/utils/prisma";⏎import·{·ADMIN_ACCOUNT_ID·}·from·"@/utils/constants` with `ADMIN_ACCOUNT_ID·}·from·"@/utils/constants";⏎import·{·prisma·}·from·"@/utils/prisma`

(prettier/prettier)